packages/components/src/TooltipTrigger/TooltipTrigger.component.js

```diff
@@ -78,7 +78,6 @@ class TooltipTrigger extends React.Component {
 				placement={this.props.tooltipPlacement}
 				overlay={tooltip}
 				delayShow={400}
-				onExited={this.onMouseOut}
```

Comment:
the bind of this.onMouseOut id removed but the function attached to the component remain and is unused.